src/apps/pstake/binance-smart-chain/pstake.Stake.token-fetcher.ts

```diff
@@ -0,0 +1,74 @@
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+import { ContractType } from '~position/contract.interface';
+import { PstakeContractFactory } from '../contracts';
+import { PSTAKE_DEFINITION } from '../pstake.definition';
+import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
+import { getTokenImg } from '~app-toolkit/helpers/presentation/image.present';
+
+
+
+const appId = PSTAKE_DEFINITION.id;
+const groupId = PSTAKE_DEFINITION.groups.stake.id;
+const network = Network.BINANCE_SMART_CHAIN_MAINNET;
+
+@Register.TokenPositionFetcher({ appId, groupId, network })
+export class BinanceSmartChainPstakeStakeTokenFetcher implements PositionFetcher<AppTokenPosition> {
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
+    @Inject(PstakeContractFactory) private readonly pstakeContractFactory: PstakeContractFactory,
+  ) { }
+
+  async getPositions() {
+    const address = '0xc2e9d07f66a89c44062459a47a0d2dc038e4fb16';
+    const multicall = this.appToolkit.getMulticall(network);
+    const contract = this.pstakeContractFactory.stakedBnbToken({ address, network });
+
+    const baseTokenDependencies = await this.appToolkit.getBaseTokenPrices(network);
+    await new Promise(r => setTimeout(r, 2000));
```

Comment:
Remove this promise wait

Reply:
Done

Reply:
@immasandwich any more changes?

Reply:
> Replace the `assets/pSTAKElogo.png` file with `logo.png` like https://s2.coinmarketcap.com/static/img/coins/200x200/15996.png

Done

Reply:
@immasandwich @kafitz @maximebedard @mikeshultz